css/global.css

```diff
@@ -724,9 +808,78 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 }
 
 /* Tablet and smaller 768px */
-@media screen and (max-width: var(--mq-width-tablet-max)) {
+@media screen and (max-width: 768px) {
```

Comment:
not sure why css vars not working with media query.  I have been trying to use --mq-width-tablet-max, but it does not register during tests.

I left in static values for now 🤔